R/access-gsplot.R

```diff
@@ -72,4 +72,28 @@ logged.gsplot <- function(object, side=NULL){
     lapply(views(object), function(x) x$window$log) %>% 
       setNames(names)
   }
+}
+
+#' Get view information from a gsplot object
+#' 
+#' get the views in gsplot object
+#' 
+#' @param object a gsplot object
+#' @return data frame with one row per view. Each view has an x side, y side, the log command, and the view index.
+#' @export
+getViewInfo <- function(object){
```

Comment:
Sorry, hard habit to break. I'll quick change it. We are doing _ for functions and . for arguments, right?